bucket/anaconda3.json

```diff
@@ -21,7 +21,7 @@
             "Move-Item \"$dir\\setup.exe\" \"$dir\\..\\setup.exe\" | Out-Null",
```

Comment:
A recent merged #10830 about 2 days ago was done that added this line. Let's see what is the rationale for moving the setup.exe to another path. If this line is needed then your PR is also needed.

Reply:
Ref: https://github.com/ScoopInstaller/Extras/issues/10829

Reply:
I don't know why but moving `setup.exe` to the parent directory did solve my installation issue. The `setup.exe` suddenly ends after verifying 100% before. #10829

Reply:
I am not sure where we discuss the issue on too many references